documentation-site/static/examples/side-navigation/basic.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import {Navigation} from 'baseui/side-navigation';
+import nav from './data.js';
```

Comment:
should we include this in this file instead of referencing this? the reader of the documentation site would have no way to open this from the site

also, currently all examples are self-contained on the doc site, so I'd add it here to be consistent too

Reply:
Agreed. It's nice to have examples self-contained even if it means some copy&pasting.